src/Sylius/Behat/Context/Api/Admin/ManagingAdministratorsContext.php

```diff
@@ -333,7 +333,7 @@ public function iShouldBeNotifiedThatItCannotBeDeleted(): void
             'Administrator could be deleted'
         );
         Assert::same(
-            $this->responseChecker->getError($this->client->getLastResponse()),
+            $this->responseChecker->getValue($this->client->getLastResponse(), 'message'),
```

Comment:
Why did you change that?

Reply:
Because its working with collections but not with items